src/pyrokinetics/gk_code/tglf.py

```diff
@@ -255,44 +255,43 @@ def get_local_geometry_miller(self) -> LocalGeometryMiller:
 
         return miller
 
-    def get_local_geometry_miller_turnbull(self) -> LocalGeometryMillerTurnbull:
+    def get_local_geometry_mxh(self) -> LocalGeometryMXH:
         """
-        Load miller_turnbull object from TGLF file
+        Load mxh object from TGLF file
         """
 
-        miller_turnbull_data = default_miller_turnbull_inputs()
+        mxh_data = default_mxh_inputs()
 
         for (pyro_key, tglf_key), tglf_default in zip(
-            self.pyro_tglf_miller_turnbull.items(),
-            self.pyro_tglf_miller_turnbull_defaults.values(),
+            self.pyro_tglf_mxh.items(),
+            self.pyro_tglf_mxh_defaults.values(),
         ):
-            miller_turnbull_data[pyro_key] = self.data.get(tglf_key, tglf_default)
+            mxh_data[pyro_key] = self.data.get(tglf_key, tglf_default)
 
-        miller_turnbull_data["s_delta"] = self.data.get("s_delta_loc", 0.0) / np.sqrt(
-            1 - miller_turnbull_data["delta"] ** 2
+        mxh_data["s_delta"] = self.data.get("s_delta_loc", 0.0) / np.sqrt(
+            1 - mxh_data["delta"] ** 2
         )
 
-        miller_turnbull_data["shat"] = (
-            self.data.get("q_prime_loc", 16.0)
-            * (miller_turnbull_data["rho"] / miller_turnbull_data["q"]) ** 2
+        mxh_data["shat"] = (
+            self.data.get("q_prime_loc", 16.0) * (mxh_data["rho"] / mxh_data["q"]) ** 2
         )
 
-        miller_turnbull_data["ip_ccw"] = 1
-        miller_turnbull_data["bt_ccw"] = 1
+        mxh_data["ip_ccw"] = 1
+        mxh_data["bt_ccw"] = 1
 
         beta = self.data.get("betae", 0.0)
-        miller_turnbull_data["B0"] = 1 / beta**0.5 if beta != 0 else None
+        mxh_data["B0"] = 1 / beta**0.5 if beta != 0 else None
 
-        miller_turnbull_data["beta_prime"] = (
+        mxh_data["beta_prime"] = (
             self.data.get("p_prime_loc", 0.0)
-            * miller_turnbull_data["rho"]
-            / miller_turnbull_data["q"]
+            * mxh_data["rho"]
+            / mxh_data["q"]
             * (8 * np.pi)
         )
 
-        miller_turnbull = LocalGeometryMillerTurnbull.from_gk_data(miller_turnbull_data)
+        mxh = LocalGeometryMillerMXH.from_gk_data(mxh_data)
```

Comment:
LocalGeometryMillerMXH --> LocalGeometryMXH

Reply:
How has this been running so far...